src/components/editor/GroupChoiceModal.jsx

```diff
@@ -28,18 +24,13 @@ const GroupChoiceModal = (props) => {
   }
 
   const saveAndClose = () => {
-    props.saveStarted()
-    const request = publishRDFResource(props.currentUser, props.rdf(), selectedValue)
+    props.publishMyResource(props.currentUser, selectedValue)
+
+    if (props.error) {
+      alert(props.error)
+      console.error(props.error)
```

Comment:
I was trying to figure out if we do this sort of reporting back to the user when retrieving an existing resource, and it didn't appear to be true to me ... did I misunderstand the code?